config/settings.py

```diff
@@ -148,3 +148,16 @@
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'frontend', 'src', 'static'),
 )
+
+STATIC_ROOT = os.path.join(BASE_DIR, 'assets')
+
+REST_FRAMEWORK = {
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
+    'PAGE_SIZE': 10
+}
+
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'mahmoud@alminawi.org'
```

Comment:
what is this and do we need a testing email as a placeholder

Reply:
I did put the email as a placeholder to test. I did create a gmail testing account so my email won't be there.